src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figures {
+    private int side;
+    private int upSide;
+    private int downSide;
+
+    public IsoscelesTrapezoid(String color, int side, int upSide, int downSide) {
+        super(color);
+        this.side = side;
+        this.upSide = upSide;
+        this.downSide = downSide;
+    }
+
+    @Override
+    public double area() {
+        return side * 2 + upSide + downSide;
+    }

```

Comment:
The formula for the area of an isosceles trapezoid is incorrect. The correct formula is ((upSide + downSide) / 2) * height. Please check if you have a height parameter for your trapezoid, if not, you should add it.